packages/contracts/test/framework/dao/dao-factory.ts

```diff
@@ -217,9 +205,7 @@ describe('DAOFactory: ', function () {
     );
 
     // Deploy DAO Factory
-    const DAOFactory = new ethers.ContractFactory(
-      mergedABI,
-      daoFactoryBytecode,
+    const DAOFactory = new DAOFactory__factory(
       signers[0]
     ) as DAOFactory__factory;
```

Comment:
typecasting isn't necessary anymore

Reply:
Fixed in 8f6b669878e33c0d6667e9a0c358c6cf8a156920